React/tic-tac/src/app/screens/Game/index.js

```diff
@@ -41,7 +46,7 @@ class Game extends Component {
   mapMove = (step, move) => {
     const desc = `${move ? `Go to move #${move}` : 'Go to game start'}`;
     return (
-      <li key={move}>
+      <li className={style.listMoveButton} key={move}>
         <button className={style.showMove} onClick={() => this.jumpTo(move)}>
```

Comment:
Do not define functions within JSX syntax since it may trigger unwanted re-renderers